dist/src/main/config/broker.standalone.yaml.template

```
@@ -524,6 +524,19 @@
         # This setting can also be overridden using the environment variable ZEEBE_BROKER_EXPERIMENTAL_RAFT_REQUESTTIMEOUT
         # requestTimeout = 5s
 
+        # If the leader is not able to reach the quorum, the leader may step down.
+        # This is triggered after minStepDownFailureCount number of requests fails to get a response from the quorum of
+        # followers as well as if the last response was received before maxQuorumResponseTime.
```

I have the feeling we can improve this sentence somehow.
```suggestion
        # This is triggered after a number of requests, to a quorum of followers, has failed, and we reached 
        # minStepDownFailureCount. The maxQuorumResponseTime plays here a certain role, the requests need to 
       # be successful before that timeout.
```

If you agree then please adjust the other comment as well.

---

I have updated the description. I have tried to make it clear the meaning of maxQuorumResponseTime. Please check it again if you like. Otherwise I will merge it later today. :timer_clock: